CONTRIBUTING.md

```diff
@@ -37,7 +35,16 @@ This is a rough outline of what a contributor's workflow looks like:
 - Make sure your commit messages are in the proper format (see below).
 - Push your changes to a topic branch in your fork of the repository.
 - Make sure the tests pass, and add any new tests as appropriate.
-- Submit a pull request to the original repository.
+- Submit a pull request to the original repository and assign it to yourself.
```

Comment:
I think you're right, will recheck the GitHub settings, otherwise remove the line.